src/Microsoft.AspNetCore/project.json

```diff
@@ -0,0 +1,27 @@
+﻿{
+  "version": "1.0.1-*",
+
+  "dependencies": {
+    "NETStandard.Library": "1.6.0",
+
+    "Microsoft.AspNetCore.Diagnostics": "1.0.0",
+    "Microsoft.AspNetCore.Hosting": "1.0.0",
+    "Microsoft.AspNetCore.StaticFiles": "1.0.0",
+
+    "Microsoft.Extensions.Configuration.EnvironmentVariables": "1.0.0",
+    "Microsoft.Extensions.Configuration.Json": "1.0.0",
+    "Microsoft.Extensions.Logging": "1.0.0",
+    "Microsoft.Extensions.Logging.Console": "1.0.0",
+    "Microsoft.Extensions.Options.ConfigurationExtensions": "1.0.0",
+    "Microsoft.Extensions.Configuration.FileExtensions": "1.0.0",
+
+    "Microsoft.AspNetCore.Routing": "1.0.1",
```

Comment:
This is how they were grouped in the spec. I think this might be fine for this package because it's... different. More of a logical grouping.